source/includes/_intro.md

```diff
@@ -394,25 +394,267 @@ The first step in most situations will be to instantiate a client. (TODO these c
+done by listing them, or in a calendar interface, or in any other way that is
+appropriate for your application.
+
+**Representative Test:** User selects *The Unremarkable Incident of the Cat at Lunchtime*
+(event ID 7AB)<br/>
+When the event is selected in the interface,<br/>
+Then it should display the two available performances, 1 January {this year + 1}
+and 1 January {this year + 2}, at 15:30:00 UTC.
+
+### Support best available booking flow
+At a minimum, the customer must be able to specify the number of seats they
+want and for which ticket type and price band. The Ingresso backend will then
+reserve the best available seats that meet the requirements.
+
+**Representative Test:** User selects 3 tickets for event 7AB in the stalls band B<br/>
+When the reservation is made<br/>
+Then the application shows a reservation confirmation screen with 3 tickets from
+that price band (such as D2, D3 and D4), and allows the user to proceed with
+the booking and payment process
+
+### Display the full ticket price to the customer before purchase
+The customer must be shown the full price they will pay before the purchase is
+confirmed and their payment taken.
+<aside class="notice">If selling in the UK market, the ticket seat price and
+the surcharge should be itemised separately. Make sure to follow all market
+regulations for the countries and regions you are selling in.</aside>
+
+**Representative Test:** User confirms reservation of 1 seat for event 7AB in the stalls
+band A (e.g. seat B4)<br/>
+When the confirmation and payment screen is displayed<br/>
+Then the full price should be displayed as £55.00, showing a £50.00 seat price
+and a £5.00 surcharge. Any dispatch methods that incur an additional cost
+should also be listed separately. A total price should be displayed in a
+highlighted font showing the price the customer will pay (by adding the prices
+together).
+
+### Display special seat conditions to customer before purchase
+Some seats in venues may have restricted views of the stage, or other
+information associated with them. It's important that the customers are made
+aware of any special seat conditions that will apply before they purchase.
+
+**Representative Test:** User confirms reservation of 3 seats in CIRCLE-B for
+event 7AB including seats F1-F3<br/>
+When the reservation is made<br/>
+Then the confirmation screen should show that these seats have restricted
+views, and the seat text for F2.
+
+### Release seats if customer does not proceed with the booking flow
+A customer may change their mind about the number or type of seats to be
+reserved for a performance. If your customer selects and reserves seats, then
+goes back to select and reserve additional seats or removes the reserved seats
+from their basket, the initial seats should be [released](#release).
+
+Failure to release the initial reservation will mean the customer is unable to
+reserve seats (even though they are actually available). Ingresso will
```

Comment:
Suggest: "unable to reserve the initial seats (even though they are actually available), and needlessly reduces inventory for everyone."